frontend-react/src/pages/Upload.tsx

```diff
@@ -53,3 +57,7 @@ export const Upload = () => {
         </div>
     );
 };
+
+export const UploadWithAuth = () => (
+    <AuthElement element={Upload} requiredUserType={MemberType.SENDER} />
```

Comment:
[thought for future change] since this page isn't functional anymore, does it still need to be behind auth?